wp-session-manager.php

```diff
@@ -33,3 +33,30 @@
 	include 'includes/class-wp-session.php';
 	include 'includes/wp-session.php';
 }
+
+// Create the required table.
+add_action('admin_init', 'create_sm_sessions_table' );
+function create_sm_sessions_table() {
+	$current_db_version = '0.1';
+	$created_db_version = get_option('sm_session_db_version', '0.0' );
+
+	if ( version_compare( $created_db_version, $current_db_version, '<' ) ) {
+		global $wpdb;
+
+		$collate = '';
+		if ( $wpdb->has_cap( 'collation' ) ) {
+			$collate = $wpdb->get_charset_collate();
+		}
+
+		$table = "CREATE TABLE {$wpdb->prefix}sm_sessions (
```

Comment:
I'm OK with the table name, but this table doesn't seem to store any session data. It keeps track of the session ID and key and expiry, which is great, but there's no column for any serialized session information (what used to be stored in the `_wp_session_{$id}` option).

Reply:
@ericmann I've updated the code and should now store the container value. I've also refactored some functions, fixed some bugs and updated comments. Let me know if i've missed anything.